traitsui/testing/tester/qt4/implementation/instance_editor.py

```diff
@@ -0,0 +1,51 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+from traitsui.testing.tester.registry_helper import register_nested_ui_solvers
+from traitsui.qt4.instance_editor import (
+    CustomEditor,
+    SimpleEditor
+)
+
+
+def _get_netsed_ui_simple(target):
+    """ Obtains a nested UI within a Simple Instance Editor.
+
+    Parameters
+    ----------
+    target : instance of SimpleEditor
+    """
+    if target._dialog_ui is None:
+        target._button.click()
```

Comment:
Now these two lines can be removed as they are redundant in tests where a mouse click is performed prior to querying for the dialog UI.  I am sorry I probably should have done that in #1206 😓 .

If the test code tries to navigate into the dialog UI without performing a mouse click first, the dialog UI is None and we get an error anyway complaining no location solvers are registered on `None`. It might be nicer to raise early with a different error message but I think it might be a bit overkill.